builder/src/test/java/dev/sashimono/builder/complete/BuildProjectTestCase.java

```diff
@@ -23,12 +23,14 @@ public void testBuildingSimpleProject(BuildResult result) throws IOException {
         Path jar = dir.resolve("test-1.1.0.Final.jar");
         Path pom = dir.resolve("pom.xml");
         Path sourcesJar = dir.resolve("test-1.1.0.Final-sources.jar");
+        Path javadocJar = dir.resolve("test-1.1.0.Final-javadoc.jar");
         Assertions.assertTrue(Files.exists(jar));
         Assertions.assertTrue(Files.exists(pom));
         Assertions.assertTrue(Files.exists(sourcesJar));
+        Assertions.assertTrue(Files.exists(javadocJar));
         String expectedAppPropsContents = """
                 greeting.message = hello
-                greeting.name = quarkus""".replaceAll("\n", System.lineSeparator());
+                greeting.name = quarkus""".replaceAll(NEW_LINE, System.lineSeparator());
```

Comment:
It does not really matter either way, but for stuff like this using a constant is less clear and easy to read than just using "\n" (NEW_LINE is pretty  obvious, but for stuff like 'DELIMITER = ":"' if you are reading the code you have to scroll up to see what the value actually is). 

I know some people treat it as dogma that all strings must be constants, but IMHO it does not actually help at all in cases like this and actually just makes the code a bit harder to read.